.tx/config

```diff
@@ -9,7 +9,7 @@ file_filter = MapboxNavigation/Resources/<lang>.lproj/Localizable.strings
 source_file = MapboxNavigation/Resources/Base.lproj/Localizable.strings
 source_lang = en
 
-[mapbox-navigation-core-ios.localizablestrings]
+[mapbox-navigation-ios.LocalizableCoreStrings]
 file_filter = MapboxCoreNavigation/Resources/<lang>.lproj/Localizable.strings
```

Comment:
Renaming slugs seems to be working as long as we rename it both in Transifex and in the repo at the same time. Let's hope auto updating resources ™ doesn't delete any strings.